src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonBirthday/PersonBirthday.tsx

```diff
@@ -32,16 +32,27 @@ export const PersonBirthday: React.FC<PersonBirthdayProps> = ({
     setFieldValue('birthdayYear', date?.year || null);
   };
 
+  const birthdayDate =
+    birthdayMonth && birthdayDay
+      ? DateTime.local(birthdayYear ?? 1900, birthdayMonth, birthdayDay)
+      : null;
+
+  const invalidDate =
+    birthdayMonth === 0 && birthdayDay === 0
+      ? DateTime.local(birthdayYear ?? 1900, birthdayMonth, birthdayDay)
```

Comment:
Seems like this duplication could be factored out.

Reply:
Are you referring to the duplication as `birthdayDate` and `invalidDate` or something else?

Reply:
```
DateTime.local(
          ...Year ?? 1900,
          ...Month,
          ...Day,
        )
```
is in the code 4 times (just in this PR). The `invalidDate` logic is also duplicated between birthday and anniversary.